### PR TITLE
Fixes bugs in NormalCatalog.py

### PR DESCRIPTION
**Pull request by keikojima02**
### Summary
**Fixes the following problems:**
- When adding edges to existing nodes, code would not create/add anything.
- When data didn't give entire path of a taxonomic hierarchy for each species or other taxonomic categories, code did not return correct results.
- Node property `name:` was still including author and date.

### Related Issues

- Resolves Issue #65

### Backwards incompatibilities

None

### New Dependencies

None

### CODE REVIEW

**Comment by keikojima02** [comment]
**Result**

image

File: modules/mining/OptimizedCatalog.py [hunk 1]
```diff
@@ -15,7 +15,7 @@
 
 from .NormalCatalog import create_dir, to_json
 
-IMPORT = '../../.Neo4jDesktop/neo4jDatabases/database-328e805b-80fc-42bc-bf79-44a07e3e1447/installation-4.0.2/import/'
+IMPORT = 'C:/Users/kkojima1/.Neo4jDesktop/neo4jDatabases/database-6d93c068-d88e-470a-877e-8f5379d19246/installation-4.0.4/import/'
```
[the remark this answers is not in the document]
Comment (pjuangph) [reply modules/mining/OptimizedCatalog.py:18]: if you can change it to using a connection string that would be most ideal

File: modules/mining/OptimizedCatalog.py [hunk 2]
```diff
@@ -43,7 +43,7 @@ def to_long_json():
             # if rank == 'infraspecies':
             #     if taxon == 'species': # Necessary
             #         found = True
-            elif taxon == rank:
+            if taxon == rank:
```
[the remark this answers is not in the document]
Comment (keikojima02) [reply modules/mining/OptimizedCatalog.py:52]: Does the comments make sense? See commit bf51377198bdc55faa697a822d649fd1031cc1ca

File: modules/mining/OptimizedCatalog.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (keikojima02) [reply modules/mining/OptimizedCatalog.py:18]: Would reverting back to what Lucas had a relative connection string?

[the remark this answers is not in the document]
Comment (keikojima02) [reply modules/mining/OptimizedCatalog.py:18]: Sorry *relative path

[the remark this answers is not in the document]
Comment (keikojima02) [reply modules/mining/OptimizedCatalog.py:18]: I did create code to automatically find the import folder, but my problem is if the user has multiple databases.

[the remark this answers is not in the document]
Comment (pjuangph) [reply modules/mining/OptimizedCatalog.py:18]: This is for importing the csv files. This is not for connection to database. user will need to set this to whatever their home directory is if they want to import csv's into neo4j

**Pull request merged**